spotify_forms/main_form.py

```diff
@@ -1,19 +1,12 @@
-from browser.py_quality_services import PyQualityServices
-from forms.base_form import BaseForm
-from selenium.webdriver.common.by import By
+from playwright.sync_api import Page
 
 
-class MainContentForm(BaseForm):
-    __search_input = PyQualityServices.element_factory.get_text_box((By.XPATH, "//form[@role='search']//input"),
-                                                                    "Search input")
+class MainContentForm:
 
-    def __init__(self):
-        super(MainContentForm, self).__init__((By.XPATH, "//div[@class='main-view-container']"),
-                                              name="Main form")
+    __search_input_xpath = "xpath=//form[@role='search']//input"
 
-    def search_singer(self, name):
-        self.__search_input.send_keys(name)
+    def search_singer(self, page: Page, name):
+        page.locator(self.__search_input_xpath).fill(name)
```

Comment:
Can we have a singleton or threadsafe container for page instance? in order not to pass it to each form method